libs/components/layout/testing/src/modules/text-expand/text-expand-harness.ts

```diff
@@ -0,0 +1,63 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+
+import { SkyTextExpandHarnessFilters } from './text-expand-harness-filters';
+import { SkyTextExpandModalHarness } from './text-expand-modal-harness';
+
+/**
+ * Harness for interacting with a text expand component in tests.
+ */
+export class SkyTextExpandHarness extends SkyComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = 'sky-text-expand';
+
+  #documentRootLocator = this.documentRootLocatorFactory();
+  #getExpandCollapseButton = this.locatorForOptional(
+    '.sky-text-expand-see-more',
+  );
+  #getText = this.locatorFor('.sky-text-expand-text');
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyTextExpandHarness` that meets certain criteria.
+   */
+  public static with(
+    filters: SkyTextExpandHarnessFilters,
+  ): HarnessPredicate<SkyTextExpandHarness> {
+    return SkyTextExpandHarness.getDataSkyIdPredicate(filters);
+  }
+
+  /**
+   * Clicks the button element to expand or collapse text.
+   */
+  public async clickSeeMoreButton(): Promise<void> {
```

Comment:
this would either be `see more` or `see less` right? i think this can be named differently. but im kinda stumped on what it could be 😓 @blackbaud-johnly any thoughts?

Reply:
We call the interla function here `getExpandCollapseButton` - could we just name the public one `clickExpandCollapseButton`?